HarmonyCore.Test/EF/MultXfConn.dbl

```diff
@@ -62,55 +63,55 @@ namespace HarmonyCore.Test.EF
 			data scsportsts, int
 			xcall setlog("SCSPORT", %atrimtostring(scsport), scsportsts)
 		endmethod
+                
+        private delegate LoadInfoTestHelperDelegate, @Action
+            channelManager, @FileChannelManager
+        enddelegate
+        
+        private method LoadInfoTestHelper, @Action
+            channelManager, @FileChannelManager
+        proc
+            lambda openFiles()
+            begin
+                data file1 = new IsamDataObjectIO<Orders>(channelManager, "DAT:orders.ism", FileOpenMode.UpdateIndexed)
+                data lid1 = FileChannelManager.LoadInfo.CurrentLoadId
+                data file2 = new IsamDataObjectIO<Customer>(channelManager, "DAT:customers.ism", FileOpenMode.UpdateIndexed)
+                data lid2 = FileChannelManager.LoadInfo.CurrentLoadId
+                data file3 = new IsamDataObjectIO<Orders>(channelManager, "TESTING1:orders.ism", FileOpenMode.UpdateIndexed)
+                data lid3 = FileChannelManager.LoadInfo.CurrentLoadId
+                data file4 = new IsamDataObjectIO<Customer>(channelManager, "TESTING1:customers.ism", FileOpenMode.UpdateIndexed)
+                data lid4 = FileChannelManager.LoadInfo.CurrentLoadId
+                Assert.AreEqual(lid1, lid2)
+                Assert.AreEqual(lid3, lid4)
+            end
+            mreturn openFiles
+        endmethod
 
         {TestMethod}
         public method LoadInfoTest, void
         proc
-            data xfconnloadsts, int
-            xcall setlog("HC_XFCONNECTIONLOAD", 2, xfconnloadsts)
-            disposable data channelManager = new FileChannelManager()
+            data sts, int
+            xcall setlog("HC_XFCONNECTIONLOAD", 2, sts)
+            xcall setlog("HC_XFCONNECTIONBALANCING", "", sts)
+            data channelManager = new FileChannelManager()
+            ; set connectionLoadCount
+            data typeFCM = ^typeof(FileChannelManager)
+            data connectionLoadCountField = typeFCM.GetField("connectionLoadCount", BindingFlags.Static | BindingFlags.NonPublic)
+            data setConnectionLoadCount = typeFCM.GetMethod("SetConnectionLoadCount", BindingFlags.Static | BindingFlags.NonPublic)
+            connectionLoadCountField.SetValue(^null, setConnectionLoadCount.Invoke(^null, ^null))
+            
+            data deleg, @LoadInfoTestHelperDelegate
```

Comment:
I dont think you need deleg here, I think you can just call LoadInfoTestHelper(some_arg) anywhere you use deleg